posts/2019-05-15-4-Years-Of-Rust.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: "4 years of Rust"
+author: The Rust Core Team
+---
+
+On May 15th, 2015, [Rust][rust-release] was released to the world! After 5 years of open development (and a couple of years of sketching before that), we finally hit the button on making the attempt to create a new systems programming language a serious effort!
+
+It’s easy to look back on the pre-1.0 times and cherish them for being the wild times of language development and fun research. Features were added and cut, syntax and keywords were tried, and before 1.0, there was a big clean-up that removed a lot of the standard library. For fun, you can check Niko’s blog post on [how Rust's object system works][rust-object-system], Marijn Haverbeke’s talk on [features that never made it close to 1.0][marijn-rustfest] or even the [introductory slides about Servo][servo-introduction], which present a language looking very different from today.
+
+Releasing Rust with stability guarantees also meant putting a stop to large visible changes. The face of Rust is still very similar to Rust 1.0. Even with the last year’s 2018 Edition idioms, Rust is still very recognizable as what it was in 2015. That steadiness hides that the time of Rust’s fastest development and growth is *now*. With the stability of the language and easy upgrades as a base, a ton of new features have been built. We’ve seen a bunch of achievements in the last year:
```

Comment:
I don't think "edition idioms" is quite the right term to use here, in the context of the edition "idioms" means a particular thing.

"Even with the changes from last year's edition"?

Reply:
```suggestion
Releasing Rust with stability guarantees also meant putting a stop to large visible changes. The face of Rust is still very similar to Rust 1.0. Even with the changes from last year’s 2018 Edition, Rust is still very recognizable as what it was in 2015. That steadiness hides that the time of Rust’s fastest development and growth is *now*. With the stability of the language and easy upgrades as a base, a ton of new features have been built. We’ve seen a bunch of achievements in the last year:
```